css-canvas-medical/styles.css

```diff
@@ -0,0 +1,283 @@
+.container {
+  width: 100%;
+  margin: auto;
+  max-width: 1200px;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+.img1-canvas-marg {
+  margin-top: 30px;
+  max-width: 30%;
+}
+
+.row-3-padding {
+  padding-top: 50px;
+  padding-bottom: 70px;
+}
+
+.clin-marg {
+  padding-top: 60px;
+  margin-left: 15%;
+}
+
+h2 {
+  padding-left: 5px;
+}
+
+.font-a3 {
+  font-size: 15px;
+  padding-right: 5px;
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 550;
+}
+
+.font-avenir-normal {
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+}
+
+.font-avenir-normal-small {
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 14px;
+}
+
+.font-avenir-normal-medium {
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+}
```

Comment:
If you apply your `font-family` to the `body` one time, you won't have to apply it to separate elements. Also, I'm pretty sure `font-style` is `normal` by default, so there's no need to add it.